Description: Enable usage of PyCXX on Py3k too
 Given how PyCXX has been packaged in Debian, we can use the py2 and py3 version
 with matplotlib.
Author: Sandro Tosi <morph@debian.org>
Origin: vendor
Bug-Debian: https://bugs.debian.org/613818
Forwarded: not-needed
Last-Update: 2015-02-11
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/setupext.py
+++ b/setupext.py
@@ -840,15 +840,15 @@ class CXX(SetupPackage):
     name = 'pycxx'
 
     def check(self):
-        if PY3:
-            # There is no version of PyCXX in the wild that will work
-            # with Python 3.x and matplotlib, since they lack support
-            # for the buffer object.
-            self.__class__.found_external = False
-            return ("Official versions of PyCXX are not compatible "
-                    "with matplotlib on Python 3.x, since they lack "
-                    "support for the buffer object.  Using local "
-                    "copy")
+        #if PY3:
+        #    # There is no version of PyCXX in the wild that will work
+        #    # with Python 3.x and matplotlib, since they lack support
+        #    # for the buffer object.
+        #    self.__class__.found_external = False
+        #    return ("Official versions of PyCXX are not compatible "
+        #            "with matplotlib on Python 3.x, since they lack "
+        #            "support for the buffer object.  Using local "
+        #            "copy")
 
         self.__class__.found_external = True
         old_stdout = sys.stdout
